assets/js/steps/bibbox.spec.js

```diff
@@ -5,157 +5,183 @@
 
 import React from 'react';
 import Bibbox from './bibbox';
-import { shallow, mount } from 'enzyme';
+import { mount } from 'enzyme';
 import { expect, it, describe } from '@jest/globals';
+import { IntlProvider } from 'react-intl';
 
-it('renders without crashing', () => {
-    shallow(
-        <Bibbox
-            boxConfigurationInput={{
-                inactivityTimeOut: 3000,
-                school: {
-                    name: 'Mårslet Skole'
-                }
-            }}
-            machineStateInput={{ step: 'initial' }}
-            actionHandler={() => { }}
-        />
-    );
-});
+const translations = {
+    'initial-choose-a-function': 'Select a function to start',
+    'initial-button-check-out': 'Loan',
+    'initial-button-status': 'Status',
+    'initial-button-check-in': 'Hand in',
+    'login-not-configured': 'Login method is not configured',
+    'status-header-current-loans': 'Current loans',
+    'status-header-reservations': 'Reservations',
+    'status-header-ready-for-pickup': 'Ready for pickup',
+    'banner-header-book-with-fine': 'This book has a fine',
+    'banner-heaeder-book-for-check-in': 'This book must be handed in',
+    'help-box-header': 'Help',
+    'book-is-registered': 'The book has been registered. Ready for the next one',
+    'button-navbar-check-out': 'Loan',
+    'button-navbar-status': 'Status',
+    'button-navbar-check-in': 'Hand in',
+    'button-navbar-finish': 'Exit'
+};
+
+// it('renders without crashing', () => {
```

Comment:
Remove commented out code, if it is not relevant anymore

Reply:
Seams to be outdated. Fixed.